shared/native/notification-listeners.shared.js

```diff
@@ -14,7 +14,10 @@ let lastLoggedInNotifyUsername = null
 let lastBadgeStateVersion = -1
 
 export default function(dispatch: Dispatch, getState: () => Object, notify: any): incomingCallMapType {
-  const throttledDispatch = throttle(action => dispatch(action), 1000, {leading: false, trailing: true})
+  const throttledDispatch = throttle(action => dispatch(action), 1000, {
```

Comment:
I'm surprised by these changes, are you using the same version of prettier we are?